src/utils/formatter.js

```diff
@@ -135,15 +135,28 @@ export const formatCardNumber = cardNumber => {
 export const formatCvc = cvc => {
   return (cvc.match(/\d+/g) || []).join('');
 };
-export const formatExpiry = prevExpiry => {
+export const formatExpiry = event => {
+  const eventData = event.nativeEvent && event.nativeEvent.data;
+  const prevExpiry = event.target.value.split(' / ').join('/');
+
   if (!prevExpiry) return null;
   let expiry = prevExpiry;
   if (/^[2-9]$/.test(expiry)) {
     expiry = `0${expiry}`;
   }
+
+  if (prevExpiry.length === 2 && +prevExpiry > 12) {
```

Comment:
`+`? I've never seen this before! 👀

Reply:
it's a way to convert a string to an integer. By doing so, in case someone is pasting letters rather numbers, that condition still working.

```js
console.log(+"13" > 12)
console.log(+"12" > 12)
console.log(+"12a" > 12)
```

Reply:
am planning to send other PRs later on... one of them is for testing and code coverage